README.md

```diff
@@ -148,6 +148,7 @@ json ex1 = json::parse(R"(
 )");
 
 // Using user-defined (raw) string literals
+using namespace nlohmann::literals;
```

Comment:
Just to make sure: existing code will compile without this `using`, because `JSON_GlobalUDLs` is `ON` by default?

Reply:
Yes.

Reply:
Thanks. Once the conflicts are resolved, we're ready to go.

Reply:
Done.